pkg/controller/perconaservermongodb/psmdb_controller.go

```diff
@@ -718,6 +724,91 @@ func (r *ReconcilePerconaServerMongoDB) deleteMongosIfNeeded(cr *api.PerconaServ
 	return r.deleteMongos(cr)
 }
 
+func (r *ReconcilePerconaServerMongoDB) reconcileConfigMap(cr *api.PerconaServerMongoDB, repls []*api.ReplsetSpec) error {
+	for _, rs := range repls {
+		configMap := &corev1.ConfigMap{
```

Comment:
it's not needed to create object if `rs.Configuration` is empty

Reply:
changed in 885c4a7